sds_data_manager/lambda_images/instruments/dependents.json

```diff
@@ -0,0 +1,196 @@
+{
+    "CodiceHi": {
```

Comment:
Changed so that directories in bucket are lowercase. Will revisit once we have completely finalized naming. I might just have each instrument lead look through this. I'll put that as a todo and note it is really only a first pass attempt.